packages/design-system/src/components/primitives/dnd/use-drop.ts

```diff
@@ -248,21 +282,41 @@ export const useDrop = <Data>(props: UseDropProps<Data>): UseDropHandlers => {
 
     return {
       handleMove(pointerCoordinates) {
-        state.current.pointerCoordinates = pointerCoordinates;
+        if (latestProps.current.emulatePointerAlwaysInRootBounds === true) {
+          const rect = (rootRef.current as Element).getBoundingClientRect();
+          const padding = 2;
```

Comment:
where is this number coming from? does it depend on CSS?

Reply:
It's defined in a comment above:

```
// The pointer will be always at least 2px away from any edge of the root.
emulatePointerAlwaysInRootBounds?: boolean;
```

Needed simply to account for literal "edge"-cases.

Maybe I should add another threshold option for this.

Reply:
oh lets make it a constant with a comment, I think I missed it because I was looking at a diff